### PR TITLE
amp-script: Add TodoMVC example

### PR DESCRIPTION
**Pull request by choumx**
And move existing example to `examples/amp-script/`.

### CODE REVIEW

File: build-system/config.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ module.exports = {
     // Files in this testdata dir are machine-generated and are not part
     // of the AMP runtime, so shouldn't be checked.
     '!extensions/amp-a4a/*/test/testdata/*.js',
-    '!examples/*.js',
+    '!examples/**/*',
```
Comment (choumx) [inline build-system/config.js]: @rsimha FYI

Comment (rsimha) [reply build-system/config.js]: Gotcha. Not sure why we were running presubmit checks on `examples` in the first place.

File: build-system/config.js [hunk 2]
```diff
@@ -140,7 +140,7 @@ module.exports = {
...
     '!examples/visual-tests/**/*',
```
Comment (rsimha) [inline build-system/config.js]: This line can go away since the one above encompasses it, correct?

Comment (choumx) [reply build-system/config.js]: True, will remove in a follow-up.

**Pull request merged**